policyengine_us/tests/policy/baseline/gov/states/ar/tax/income/ar_income_tax_before_non_refundable_credits_joint.yaml

```diff
@@ -47,18 +51,9 @@
   absolute_error_margin: 1
   input:
     ar_taxable_income_joint: 25_000
-    ar_low_income_tax_joint: 1_000
+    ar_low_income_tax_joint: 500
+    ar_low_income_tax_eligible: true
     state_code: AR
   output:
     # 562.366 + round((25_000-24_301) * 0.049) = 596
-    ar_income_tax_before_non_refundable_credits_joint: 596
-
-- name: AR income tax rates - taxable income for income over $25,000
-  period: 2023
-  absolute_error_margin: 1
-  input:
-    ar_taxable_income_joint: 25_300
-    ar_low_income_tax_joint: 1_000
-    state_code: AR
-  output:
-    ar_income_tax_before_non_refundable_credits_joint: 591
+    ar_income_tax_before_non_refundable_credits_joint: 500
```

Comment:
why do we change the value from 1_000 to 500 here?

Reply:
I think it's because we change the logic of the return value, since now we choose between the low income tax and the main tax according to the eligibility, originally it is not. But according to the current logic of the formula, it seems all right.